lighthouse-core/test/lib/i18n/locales-test.js

```diff
@@ -18,13 +18,14 @@ describe('locales', () => {
     const deprecatedCodes = {
```

Comment:
based on locales.js.. it looks like there's also these ones:

'bs': 'hr',
'ln': 'fr',
'nb': 'no',
'tl': 'fil',


this list is spelled out concretely somewhere in the CLDR, too

Reply:
> based on locales.js.. it looks like there's also these ones:

the ICU in Chrome Canary (V8 7.7) and Node 12.6 (V8 7.5) stil have these as canonical locales. The intent of the test was just to double check that if someone asks for one of the locales we have strings for that they'll actually get it, and `deprecatedCodes` was to handle the awkward time when a locale is deprecated on the bleeding edge but not in our earliest supported Chrome/Node.

Since they aren't deprecated in the latest version of V8/Intl yet, and since the switch to deprecated is a nice trigger to double check our locales here, I think we should leave these out for now